routes/contribute/pad/render.js

```diff
@@ -1,5 +1,5 @@
-const { followup_count, modules, engagementtypes, metafields, map, DB, ownDB } = include('config/')
```

Comment:
removing unused elements from object deconstructing. there are 29 instances of this in the public/js folder but I didn't change that. might be we want to keep them around for later. note, removing them doesn't change the behavior since it doesn't check whether the field actually was part of an object (this is different to array deconstruction where it matters)